packages/unleash-docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:5.8
```

Comment:
bah, for some reason, if i change this to node:6 the migration script fails with the following error:

```
web_1 | > unleash-docker@1.0.0 db-migrate /
web_1 | > db-migrate up
web_1 | 
web_1 | module.js:442
web_1 |     throw err;
web_1 |     ^
web_1 | 
web_1 | Error: Cannot find module 'mkdirp'
web_1 |     at Function.Module._resolveFilename (module.js:440:15)
web_1 |     at Function.Module._load (module.js:388:25)
web_1 |     at Module.require (module.js:468:17)
web_1 |     at require (internal/module.js:20:19)
web_1 |     at Object.<anonymous> (/node_modules/db-migrate/bin/db-migrate:7:14)
web_1 |     at Module._compile (module.js:541:32)
web_1 |     at Object.Module._extensions..js (module.js:550:10)
web_1 |     at Module.load (module.js:458:32)
web_1 |     at tryModuleLoad (module.js:417:12)
web_1 |     at Function.Module._load (module.js:409:3)
```